### PR TITLE
feat(controller): added 2fa

### PR DESCRIPTION
**Pull request by Tbaile**
Adding support for 2FA for the controller

Ref:
- https://github.com/NethServer/nethsecurity/issues/887

### CODE REVIEW

File: src/components/TwoFaComponent.vue
Comment (andre8244) [inline src/components/TwoFaComponent.vue:1]: I'd rename this component to something like `TwoFaFormLayout`

File: src/components/standalone/account/two_fa/ConfigureTwoFaDrawer.vue
Comment (andre8244) [inline src/components/standalone/account/two_fa/ConfigureTwoFaDrawer.vue:1]: Let's put this component outside of `standalone` folder

File: src/components/standalone/account/two_fa/RevokeTwoFaModal.vue
Comment (andre8244) [inline src/components/standalone/account/two_fa/RevokeTwoFaModal.vue:1]: Let's put this component outside of `standalone` folder

File: src/lib/standalone/twoFa.ts
Comment (andre8244) [inline src/lib/standalone/twoFa.ts:1]: Same as above

File: src/views/controller/AccountView.vue [hunk 1]
```diff
@@ -20,6 +20,9 @@ import GenerateSSHKeyPairDrawer from '@/components/controller/account_settings/G
 import DeleteSSHKeyModal from '@/components/controller/account_settings/DeleteSSHKeyModal.vue'
 import { useNotificationsStore } from '@/stores/notifications'
 import ChangeLangCombobox from '@/components/ChangeLangCombobox.vue'
+import DeleteUserModal from '@/components/controller/users/DeleteUserModal.vue'
```
Comment (andre8244) [inline src/views/controller/AccountView.vue:23]: This import seems unused

File: src/views/controller/AccountView.vue [hunk 2]
```diff
@@ -20,6 +20,9 @@ import GenerateSSHKeyPairDrawer from '@/components/controller/account_settings/G
...
+import ConfigureTwoFaDrawer from '@/components/standalone/account/two_fa/ConfigureTwoFaDrawer.vue'
```
Comment (andre8244) [inline src/views/controller/AccountView.vue:24]: Ditto

File: src/views/controller/UsersView.vue
Comment (andre8244) [inline src/views/controller/UsersView.vue:1]: The only changes in this file are imports resorted? :thinking:

Comment (Tbaile) [reply src/views/controller/UsersView.vue:1]: Started working on it, added to commit then forgot to remove the changes :sweat_smile: 

I'm coming in with new commits that should view the new value provided by the API

File: src/components/TwoFaComponent.vue
Comment (Tbaile) [reply src/components/TwoFaComponent.vue:1]: Don't like that much, but in the end, probably the component is not needed.

Comment (andre8244) [reply src/components/TwoFaComponent.vue:1]: The "Component" suffix sounds a bit generic to me; anyway if you prefer we can delete the component

**Review by andre8244 — APPROVED** [review]

**Pull request merged**